lib/tokens.js

```diff
@@ -44,6 +44,9 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
     // a field that isn't part of it; however, uniqueness is handled by the
     // index below and by the randomness property of batch IDs
     collection: 'tokenization-tokenBatch',
+    // FIXME: may be able to include `tokenVersion.id` in this index, and
```

Comment:
@dlongley our efforts in getting our issues under control has given me a new perspective on FIXMEs and TODOs in code.  What are your expectations about when/if this will *ever* be resolved?

If this is a FIXME that should be resolved prior to this PR being completed, I think we need some new convention for that.

Reply:
The FIXME should be resolved before this PR is merged. I need a way to make comments like that such that they aren't lost -- when they can't be immediately implemented.